### DISCUSSION TITLE
warning message for Player????

### DISCUSSION THREAD

**Discussion by atfam** [Scythe-Anticheat/Scythe-Anticheat#146] [Help and Support]
is there a warning matches from the anti-Cheat when is detected

On that Cheater side @MrDiamond64 

Thanks again

**Comment by MrDiamond64** [comment]
The cheater that gets flagged do not get any warnings unless they have the `notify` tag.

This question is much more suited to be a Discussion post instead of an Issue, so I have transferred it here.

**Reply by atfam** [reply]
Thank you
